apps/core/src/config/local-cognito-jwt-verifier.ts

```diff
@@ -0,0 +1,35 @@
+import { CognitoJwtVerifier } from 'aws-jwt-verify';
+import { JwksCache, JwkWithKid, Jwks } from 'aws-jwt-verify/jwk';
+
+// JwksCache built for local development, works in conjunction with cognito-local
+const localJwksCache: JwksCache = {
+  getJwk: (): Promise<JwkWithKid> => Promise.resolve(localJwk),
+  getCachedJwk: (): JwkWithKid => localJwk,
+  addJwks: (): void => {
+    /** NOOP **/
+  },
+  getJwks: (): Promise<Jwks> => Promise.resolve(localJwks),
+};
+// jwk copied from https://github.com/jagregory/cognito-local/blob/master/src/keys/cognitoLocal.public.json
+const localJwk = {
+  kty: 'RSA',
+  e: 'AQAB',
+  use: 'sig',
+  kid: 'CognitoLocal',
+  alg: 'RS256',
+  n: '2uLO7yh1_6Icfd89V3nNTc_qhfpDN7vEmOYlmJQlc9_RmOns26lg88fXXFntZESwHOm7_homO2Ih6NOtu4P5eskGs8d8VQMOQfF4YrP-pawVz-gh1S7eSvzZRDHBT4ItUuoiVP1B9HN_uScKxIqjmitpPqEQB_o2NJv8npCfqUAU-4KmxquGtjdmfctswSZGdz59M3CAYKDfuvLH9_vV6TRGgbUaUAXWC2WJrbbEXzK3XUDBrmF3Xo-yw8f3SgD3JOPl3HaaWMKL1zGVAsge7gQaGiJBzBurg5vwN61uDGGz0QZC1JqcUTl3cZnrx_L8isIR7074SJEuljIZRnCcjQ',
+};
```

Comment:
Is this secure?

Is this [`// TODO:`](https://github.com/awslabs/iot-application/pull/45/files#diff-6620eba2a82214cdd85f2906cf177ec127d7032f7d9a1698739ef487fb4b5211R4) implying the long-term solution?

Reply:
this is for local dev purpose only
in production, the JWK will be specific for each user pool (we will need to add the config for production verifier)